### ISSUE TITLE
SyntaxError: Unexpected token 'export'

### ISSUE THREAD

**Issue by mtbspace** [rowanwins/smallest-enclosing-circle#6]
If I install this module using `npm install smallest-enclosing-circle --save`

And then use the minimum code to test it:
```
const enclosingCircle = require('smallest-enclosing-circle')
const test = enclosingCircle([{x: 0, y: 0}, {x: 10, y: 10}, {x: 20, y: 20}])
console.log(test);
```
The following error is output: `SyntaxError: Unexpected token 'export'`

**Comment by rowanwins** [comment]
Hi @mtbspace 

For the timebeing you'll need to use like
````
const enclosingCircle = require('smallest-enclosing-circle/dist/smallestEnclosingCircle.js')
const test = enclosingCircle([{x: 0, y: 0}, {x: 10, y: 10}, {x: 20, y: 20}])
console.log(test);
````

I need to update the `main` in the package.json to point to the built file.

**Comment by mtbspace** [comment]
Thanks @rowanwins  - that works

**Comment by rowanwins** [comment]
Have also just published 1.0.2 which should resolve things properly

**Comment by mtbspace** [comment]
Thanks @rowanwins 
Confirmed working using my original module import
